### PR TITLE
fix: post to api3.geo.admin instead of get

### PR DESCRIPTION
**Pull request by johannschwabe**
GET -> POST
The URL gets too long for routes with many points, and the request fails. Putting the geom into the body prevents this.

I needed to add the apt install to start the Docker Compose on an M1 Mac (MacOS 14). I'm not sure why that is required only for MacOS.

### CODE REVIEW

File: backend/app.py
Comment (wp99cp) [inline backend/app.py:1]: linters sind wirklich komisch, naja... dann habern wir halt noch diesen unrelated change.

File: swiss_TLM_api/local-dev.Dockerfile
Comment (wp99cp) [inline swiss_TLM_api/local-dev.Dockerfile:1]: bist du sicher, dass wir den compiler g++ brauchen?

File: backend/app.py
Comment (johannschwabe) [reply backend/app.py:1]: danke

File: swiss_TLM_api/local-dev.Dockerfile
Comment (johannschwabe) [reply swiss_TLM_api/local-dev.Dockerfile:1]: Bei mir ists nicht ohne gegangen.

**Comment by johannschwabe** [comment]
Linter mag MacOS nicht: finded Chrome nicht.

**Comment by maede97** [comment]
> Cypress mag MacOS nicht: finded Chrome nicht.
Also im Docker container drin?

**Comment by wp99cp** [comment]
> Nice! Thank you for the fix. Maybe we can split the fix into two parts: changing GET to POST as one (which should make CI actions run again happily) and getting things to work on MacOS separately (which includes chaning docker files)

The problem with the CI actions is that we are using an outdated cypress version and an outdated version of Chrome, which is no longer available. The container with the e2e test fails to build, it does not not fail run.

I tried bumping cypress to v13, but for some reason, locally the cypress container refuses to start at all on my machine. I probably need to change some configs....

**Comment by wp99cp** [comment]
> Cypress mag MacOS nicht: finded Chrome nicht.



> Nice! Thank you for the fix. Maybe we can split the fix into two parts: changing GET to POST as one (which should make CI actions run again happily) and getting things to work on MacOS separately (which includes chaning docker files)

I agree, that we could split the M2 chip compatibility issues to a new PR, but that's not the origin of the failing test cases - at least I don't think it is.

**Comment by wp99cp** [comment]
All test cases ran through successfully, but the runner crashed during cleanup. We thus should, rollback the MacOS specific changes, then squash merge that PR into dev

```
awt-cypress-1               |   7 passing (30s)
awt-cypress-1               | 
awt-cypress-1               | 
awt-cypress-1               |   (Results)
awt-cypress-1               | 
awt-cypress-1               |   ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
awt-cypress-1               |   │ Tests:        7                                                                                │
awt-cypress-1               |   │ Passing:      7                                                                                │
awt-cypress-1               |   │ Failing:      0                                                                                │
awt-cypress-1               |   │ Pending:      0                                                                                │
awt-cypress-1               |   │ Skipped:      0                                                                                │
awt-cypress-1               |   │ Screenshots:  0                                                                                │
awt-cypress-1               |   │ Video:        false                                                                            │
awt-cypress-1               |   │ Duration:     30 seconds                                                                       │
awt-cypress-1               |   │ Spec Ran:     kml.cy.js                                                                        │
awt-cypress-1               |   └────────────────────────────────────────────────────────────────────────────────────────────────┘
awt-cypress-1               | 
awt-cypress-1               | 
awt-cypress-1               | ====================================================================================================
awt-cypress-1               | 
awt-cypress-1               |   (Run Finished)
awt-cypress-1               | 
awt-cypress-1               | 
awt-cypress-1               |        Spec                                              Tests  Passing  Failing  Pending  Skipped  
awt-cypress-1               |   ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
awt-cypress-1               |   │ ✔  gpx.cy.js                                01:35       10       10        -        -        - │
awt-cypress-1               |   ├────────────────────────────────────────────────────────────────────────────────────────────────┤
awt-cypress-1               |   │ ✔  interactive_GPX.cy.js                    00:09        1        1        -        -        - │
awt-cypress-1               |   ├────────────────────────────────────────────────────────────────────────────────────────────────┤
awt-cypress-1               |   │ ✔  kml.cy.js                                00:30        7        7        -        -        - │
awt-cypress-1               |   └────────────────────────────────────────────────────────────────────────────────────────────────┘
awt-cypress-1               |     ✔  All specs passed!                        02:15       18       18        -        -        -  
```

**Review by wp99cp — APPROVED** [review]
